plugins/modules/aci_switch_policy_vpc_protection_group.py

```diff
@@ -40,6 +40,11 @@
     description:
     - The ID of the Second Leaf Switch for the Explicit vPC Protection Group.
     type: int
+  pod_id:
+    description:
+    - The pod id of the VPC member nodes.
+    type: int
+    default: 1
```

Comment:
we do not use default but mention in the description the value to which the APIC defaults to. 

and we document the range, 0 is in the docs but not sure 0 is allowed could you validate this?

```suggestion
  pod_id:
    description:
    - The pod id of the VPC member nodes.
    - Accepted values range between C(0) and C(255).
    - The APIC defaults to C(1) when unset during creation.
    type: int
```

Reply:
Understand on not using the default value, and based on the model reference, the valid values are 1 to 255, so I will update that.